src/serializer.ts

```diff
@@ -24,6 +24,15 @@ interface TransactionJSON {
   TxnSignature?: string
 }
 
+interface PaymentTransactionJSONAddition extends PaymentJSON {
+  TransactionType: 'Payment'
+}
+
+type PaymentTransactionJSON = BaseTransactionJSON &
+  PaymentTransactionJSONAddition
+
+export type TransactionJSON = BaseTransactionJSON | PaymentTransactionJSON
```

Comment:
If the `amount` and `destination` fields should be included, I think this makes sense.

If they shouldn't, maybe just add an optional `TransactionType` to the `TransactionJSON` object?

Reply:
i think they should be included, to show that to have a payment-flavored transaction you need those additional fields